packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -96,115 +86,235 @@ export async function getBalance<ReturnFormat extends DataFormat>(
 	returnFormat: ReturnFormat,
 ) {
-	const response = isHexString32Bytes(block)
-		? await rpcMethods.getUncleCountByBlockHash(web3Context.requestManager, block)
-		: await rpcMethods.getUncleCountByBlockNumber(web3Context.requestManager, block);
+	let response;
+	if (isBytes(block)) {
+		const blockHashFormatted = format(
+			{ eth: 'bytes32' },
+			block as Bytes,
+			DEFAULT_RETURN_FORMAT,
+		);
+		response = await rpcMethods.getUncleCountByBlockHash(
+			web3Context.requestManager,
+			blockHashFormatted,
+		);
+	} else {
+		const blockNumberFormatted = isBlockTag(block as string)
+			? (block as BlockTag)
+			: format({ eth: 'uint' }, block as Numbers, DEFAULT_RETURN_FORMAT);
+		response = await rpcMethods.getUncleCountByBlockNumber(
+			web3Context.requestManager,
+			blockNumberFormatted,
+		);
+	}
 
 	return format({ eth: 'uint' }, response as Numbers, returnFormat);
 }
 
 export async function getUncle<ReturnFormat extends DataFormat>(
 	web3Context: Web3Context<EthExecutionAPI>,
-	block: HexString32Bytes | BlockNumberOrTag = web3Context.defaultBlock,
-	uncleIndex: Uint,
+	block: Bytes | BlockNumberOrTag = web3Context.defaultBlock,
+	uncleIndex: Numbers,
 	returnFormat: ReturnFormat,
 ) {
-	const response = isHexString32Bytes(block)
-		? await rpcMethods.getUncleByBlockHashAndIndex(
-				web3Context.requestManager,
-				block,
-				uncleIndex,
-		  )
-		: await rpcMethods.getUncleByBlockNumberAndIndex(
-				web3Context.requestManager,
-				block,
-				uncleIndex,
-		  );
+	const uncleIndexFormatted = format({ eth: 'uint' }, uncleIndex, DEFAULT_RETURN_FORMAT);
+
+	let response;
+	if (isBytes(block)) {
+		const blockHashFormatted = format(
+			{ eth: 'bytes32' },
+			block as Bytes,
+			DEFAULT_RETURN_FORMAT,
+		);
+		response = await rpcMethods.getUncleByBlockHashAndIndex(
+			web3Context.requestManager,
+			blockHashFormatted,
+			uncleIndexFormatted,
+		);
+	} else {
+		const blockNumberFormatted = isBlockTag(block as string)
+			? (block as BlockTag)
+			: format({ eth: 'uint' }, block as Numbers, DEFAULT_RETURN_FORMAT);
+		response = await rpcMethods.getUncleByBlockNumberAndIndex(
+			web3Context.requestManager,
+			blockNumberFormatted,
+			uncleIndexFormatted,
+		);
+	}
 
 	return format(blockSchema, response as unknown as Block, returnFormat);
 }
 
 export async function getTransaction<ReturnFormat extends DataFormat>(
 	web3Context: Web3Context<EthExecutionAPI>,
-	transactionHash: HexString32Bytes,
+	transactionHash: Bytes,
 	returnFormat: ReturnFormat,
 ) {
+	const transactionHashFormatted = format(
+		{ eth: 'bytes32' },
+		transactionHash,
+		DEFAULT_RETURN_FORMAT,
+	);
 	const response = await rpcMethods.getTransactionByHash(
 		web3Context.requestManager,
-		transactionHash,
+		transactionHashFormatted,
 	);
 
 	return response === null
 		? response
 		: formatTransaction(response as unknown as Transaction, returnFormat);
 }
 
+export async function getPendingTransactions<ReturnFormat extends DataFormat>(
```

Comment:
This was not added, I just move the code up to match the order of declaration in `web3_eth.ts`